travis-ymls/aaargh.travis.yml

```diff
@@ -0,0 +1,21 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : pyarrow 
+# Source Repo         : https://github.com/apache/arrow
```

Comment:
The commit and the file name suggest that this for package "aargh", but the contents seem to be for "pyaarow", please check and update.